Story/Episode 3/np_010852.csv

```diff
@@ -0,0 +1,150 @@
+11_voice_NPC_5085_104#0,"""The current situation?<br>It's a long story so I'll just<br>summarize it for you."""
```

Comment:
This entire document has punctuation mistakes.

Also, do not preserve the linebreaks.